runtests.py

```diff
@@ -2011,6 +2011,11 @@ def flush_and_terminate(status):
 def main():
 
     global DISTDIR, WITH_CYTHON
+
+    # Set an environment variable to the top directory
+    CYTHON_PROJECT_DIR = os.path.abspath(os.path.dirname(__file__))
```

Comment:
Since this is an otherwise unused variable, I'd just assign it to the env-var, not to a local variable.

Reply:
adopted